libdevcore/UTF8.cpp

```diff
@@ -27,6 +27,50 @@
 namespace dev
 {
 
+namespace
+{
+
+/// Validate byte sequence against Unicode chapter 3 Table 3-7.
+bool isWellFormed(unsigned char byte1, unsigned char byte2)
+{
+	switch (byte1)
+	{
+	case 0xc0 ... 0xc1:
```

Comment:
gcc is complaining here that `0xc1` is beyond the value range of `unsigned char`. Perhaps make it `unsigned`?